_pages/bep_guide.md

```diff
@@ -149,6 +149,10 @@ extension proposals should embrace these rather than trying to come up with
 alternative standards. The BIDS extension proposal should therefore inventorize
 and review past and existing work that may be relevant to the BEP.
 
+### Not considering DICOM fields
+
+Many of the modalities we use have an associated standard, like DICOM for instance. While BIDS is not specifically about data format, many metadata information are stored in data files and there is rarely a good reason for using a different name than one from other established standards. In using DICOM it is reasonable to [check what DICOM has already developed](https://www.dicomstandard.org/) and see if there is overlap. In a similar fashion, when relevant, we recommend having a `source/` folder in example datasets including DICOM files (empty data but with a header, removing any personal data tags if taken from "real" data).
```

Comment:
do we mean to add a `source` folder or is this the `sourcedata` folder of BIDS?

https://bids-specification.readthedocs.io/en/latest/common-principles.html#source-vs-raw-vs-derived-data